content/en/account_management/billing/rum.md

```diff
@@ -30,3 +30,7 @@ Browser logs are automatically tied to the corresponding RUM session, enabling y
 ## Does Datadog use cookies?
 
 Yes. Datadog uses cookies to stitch together the various steps of your users into a session. This process does not use cross-domain cookies, and it does not track the actions of your users outside your applications.
+
+## Session Replays shows up in my usage page, but I have not configured Session Replay in my environment.
```

Comment:
would it be show up instead of show**s** up? @sarina-dd

Reply:
Good catch. +1 yes, this should be "show up". Also, it should be "Session replays show up", not "Session Replays" as the latter refers to what is physically seen in the UI versus the product entity itself. Will submit an edit!